test/defineFields.js

```diff
@@ -1,5 +1,5 @@
 var assert = require('assert')
-var ethUtil = require('../index.js')
+var ethUtil = require('../dist/index.js')
```

Comment:
maybe tests can be part of the babel space too for some may write these in ES6?

Reply:
Ah, yes, we can think about that, but let's leave this for now.